src/main/resources/hudson/plugins/throttleconcurrents/ThrottleJobProperty/config.jelly

```diff
@@ -20,6 +20,18 @@
              field="maxConcurrentPerNode">
       <f:textbox />
     </f:entry>
+
+    <f:optionalBlock name="limitOneJobWithMatchingParams"
+                     title="${%Prevent multiple jobs with identical parameters from running concurrently}"
+                     inline="true"
+                     checked="${instance.limitOneJobWithMatchingParams}">
+      <f:entry title="${%List of parameters to check (comma-separated)}"
+               field="paramsToUseForLimit"
+               help="${descriptor.getHelpFile('limitOneJobWithMatchingParams')}">
+        <f:textbox />
```

Comment:
Not sure what you are saying.  The help attribute is required to display the help icon.  Without it (tested) it does not display (for me at least).  Let me know what you think it should be.